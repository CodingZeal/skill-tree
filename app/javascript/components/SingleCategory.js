import React from "react";

class SingleCategory extends React.Component {
  render() {
    const { index, rating } = this.props;
    return (
      <div className="singleCategory">
        <h2 id="score">{rating.score}</h2>
        <h2 id="category_name"> {rating.category.category_name}</h2>
      </div>
    );
  }
}

export default SingleCategory;
