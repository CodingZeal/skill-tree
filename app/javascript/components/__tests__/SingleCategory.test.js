import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import SingleCategory from "../SingleCategory";

Enzyme.configure({ adapter: new Adapter() });

describe("<SingleCategorty>", () => {
  const rating = { score: 5, category: { category_name: "Purple" } };
  const wrapper = shallow(<SingleCategory rating={rating} />);
  it("renders without crashing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
