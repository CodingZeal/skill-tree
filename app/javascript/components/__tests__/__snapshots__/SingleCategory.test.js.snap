// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SingleCategorty> renders without crashing 1`] = `
<div
  className="singleCategory"
>
  <h2
    id="score"
  >
    5
  </h2>
  <h2
    id="category_name"
  >
     
    Purple
  </h2>
</div>
`;
